packages/atomic/src/pages/test.html

```diff
@@ -8,6 +8,11 @@
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
     <link rel="stylesheet" href="/themes/accessible.css" />
+    <style>
+      ::part(ripple) {
```

Comment:
Why is this added in test.html 🤔  ? 

Should this be part of all "ripple" button effect ?

Reply:
I don't understand, see my other comment, that's very specific to these tests